notebooks/data-sources/Bugzilla/bugzilla_EDA.ipynb

```diff
@@ -2,16 +2,16 @@
  "cells": [
```

Comment:
<p>Should we remove such entities? And only maybe explore the valuable ones. And one markdown we can summarize that x,y,z weren't that important? WDYT?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chauhankaranraj/ocp-ci-analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>